### PR TITLE
Define action types for adding and removing a book

### PR DESCRIPTION
**Pull request by Alfredbis29**
**add reducers and actions**

- Define action types for adding and removing a book.
- Set the initial state to be an empty array of books.
- Export Action Creators for your actions.
- Write your reducer and export it as default.
- Define state changes for the actions that you created.
- In case of unknown action - return the current state.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
   "version": "0.1.0",
   "private": true,
   "dependencies": {
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.2.0",
     "@testing-library/user-event": "^13.5.0",
+    "prop-types": "^15.8.1",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
+    "react-redux": "^8.0.1",
```
Comment (mateo951) [inline package.json:12]: react-redux is present in package.json ✔

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```
Comment (mateo951) [inline src/redux/categories/categories.js]: The value of action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux)

    const CHECK_STATUS = 'bookstore/sr/redux/categories/CHECK_STATUS'

File: src/App.js [hunk 3]
```diff
@@ -1,18 +1,26 @@
-import { Route, Routes } from 'react-router-dom';
```
Comment (mateo951) [inline src/App.js:1]: The structure of the files follows the `ducks pattern` 😉

File: src/redux/configureStore.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+import categories from './categories/categories';
+
+const root = combineReducers({ books, categories });
+
+const store = createStore(
+  root,
+  {},
+);
+
+export default store;
```
Comment (mateo951) [inline src/redux/configureStore.js:12]: * You used `combineReducers` to create a reducer 🙌
* You used `createStore` to create a store through the reducer created 🙌

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @Alfredbis29, 

📝👌 **Your project highlights** 🔥😎 
- General Requirements ✔
- Added reducers and actions 🚀

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

File: src/redux/configureStore.js (hunk 4 above)
Comment (Alfredbis29) [reply src/redux/configureStore.js:12]: Hello, @mateo951 Thanks for the code review. 
I'm not sure I understand.
 The prerequisite I require a detailed description of what has to be changed in my project.

Comment (Alfredbis29) [reply src/redux/configureStore.js:12]: @mateo951  I now get what you require me to change, and now I have made all changes possible you can now check.

**Review by dasileker — APPROVED** [review]
### **Hi @Alfredbis29,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alfredbis29** [comment]
Hello @dasileker, Thanks for your professional code review.

File: src/redux/configureStore.js (hunk 4 above)
Comment (mateo951) [reply src/redux/configureStore.js:12]: Hey @Alfredbis29, sorry for the late response! Happy to know everything went well 🔥

**Pull request merged**